code/src/components/Amount.js

```diff
@@ -0,0 +1,20 @@
+import React, {useState} from 'react'
+
+const Amount = ({choosedAmount, IselectAmount}) => {
```

Comment:
I love that you use the name of the value your sending instead of props. I am going to do the same for this week's project, super smart! 🥇